standup_notes/__main__.py

```diff
@@ -48,22 +51,32 @@ def main():
         call_func_for_specified_day(read_note, arguments)
 
     if arguments.edit:
-        call_func_for_specified_day(edit_note, arguments)
+        if arguments.copy:
+            call_func_for_specified_day(copy_prev, arguments)
+        else:
+            call_func_for_specified_day(edit_note, arguments)
 
     if arguments.copy:
         call_func_for_specified_day(copy_note, arguments)
 
+    if arguments.post:
+        matter_post()
+
 
 def call_func_for_specified_day(func, arguments):
-    """Call the proper function to either """
+    """Calls the proper function """
```

Comment:
This doesn't exactly explain what this function does. It doesn't decide what the proper function is to call, it just calls the function passed in to it, passing that function the proper day argument.

Reply:
It has been now changed to this
"""Functions calls to make sure proper day arguments are being passed to thei respective functions"""